styles/feed-event.mcss

```diff
@@ -37,6 +37,10 @@ FeedEvent {
 
   div.meta {
     font-size: 100%
-    padding: 10px
+    padding: 10px 20px
+    opacity: 0.4
+    :hover {
+      opacity: 0.8
```

Comment:
> It's hard to tell in this new design that it is a reply and not a post.

I came at this from the "everything's a post" and "all nodes are equal" thoughts, so it seemed nicer to make all posts appear visually equal, rather than make the original poster seem larger than others. 

I think we could find an icon or other visual clues that what is being shown is the middle of a discussion.

As we don't have that yet, I think using icon size is fine for now. We should find a way of visually marking replies as replies; the screenshot above is a good use-case to work on as the original post authors icon isn not visible, so it relies on the user having used the app enough to have a sense of the unseen difference (poster would be larger) rather than being explicit (oh look there is a reply icon).

All of that skips over the point that it says it in the meta... but I'm not totally convinced that the meta adds enough value to justify it's prime position at the top left of the box, but that's a conversation for later ;-)